### DISCUSSION TITLE
menu selection 구현 시 MainActivity 에서 구현하는것이 좋을까요?

### DISCUSSION THREAD

**Discussion by Gijuno** [rockphin/gyeonhae-android#45] [General]
@nutyworks 
menu selection 구현 시 MainActivity 에서 구현하는 것이 좋을까요? 아니면 Fragment를 하나 생성해서 구현하는게 좋을까요?

**Comment by nutyworks** [comment]
메뉴 선택의 경우에는 리사이클러뷰를 사용하는 것이 좋을 것 같습니다. 타이틀의 경우에는 어떻게 해야 할까요?

**Reply by Gijuno** [reply]
타이틀의 경우 visibility를 사용한 효과나 fragment를 사용하는것이 올바르다고 생각합니다만.. 둘 중 하나를 고르기가 힘드네요

**Comment by Gijuno** [comment]
타이틀 부분을 Fragment로 구현하는것이 가장 깔끔하다고 생각들어서 Fragment로 구현하겠습니다!
